templates/apis/crd.go.tpl

```diff
@@ -16,7 +16,7 @@ import (
 {{ .CRD.Documentation }}
 type {{ .CRD.Kind }}Spec struct {
 {{ range $fieldName, $field := .CRD.SpecFields }}
-{{ if $field.ShapeRef.Documentation -}}
+{{ if $field.ShapeRef -}}
```

Comment:
Can you nil-check both `ShapeRef` and `ShapeRef.Documentation` here?

Reply:
`ShapeRef.Documentation` is of type `string` (not `*string`), hence it will never cause nil pointer panics.